client/src/pages/dashboard.tsx

```diff
@@ -0,0 +1,80 @@
+import { format, fromUnixTime, getUnixTime } from 'date-fns';
+import { DashboardLayout } from '../components/DashboardLayout';
+import { Overview } from '../components/Overview';
+import { DashboardData, PriceResult } from '../types/FileData';
+import { Container } from '../ui/Container';
+
+interface DashboardProps {
+  statsData: DashboardData;
+}
+
+const Dashboard = ({ statsData }: DashboardProps) => {
+  return (
+    <Container
+      css={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '$5',
+        py: '$5',
+
+        '@lg': {
+          gap: '$20',
+          flexDirection: 'row',
+          py: '$10',
+          height: '100vh',
+        },
+
+        '@xl': {
+          gap: '128px',
+        },
+      }}
+    >
+      <DashboardLayout />
+      <Overview statsData={statsData} />
+    </Container>
+  );
+};
+
+export async function getStaticProps() {
+  const res = await fetch(process.env.DASHBOARD_API_URL!);
+  const stxStats = await res.json();
+
+  const priceRes = await fetch(process.env.STX_PRICE_API_URL!);
```

Comment:
Coingecko is limited to 8 requests/sec, so doing the call server-side would mean that our website can only server 8 requests/sec. Another approach that would work here is to call the coingecko API only on the client side, that way the IP of the user visiting the page will be used and we won't be rate-limited ✌️

Reply:
Oh I see! I never knew that - pretty awesome 😄